docs/src/infra/protocols/telnet.md

```diff
@@ -1,6 +1,144 @@
+| `status`  | Shows the current status of the telnet client                  |
+| `z`       | Suspends telnet (on Unix/Linux systems)                        |
+| `set`     | Sets Telnet options (like terminal type)                       |
+| `unset`   | Unsets Telnet options                                          |
+| `display` | Displays current settings of Telnet options                    |
+| `send`    | Sends special characters or sequences (like break)             |
+| `mode`    | Sets the mode of operation (e.g., line by line or character)   |
+| `logout`  | Logs out from the remote system (not available on all systems) |
+
+
+## Practice
+### Enumeration
+#### Banner Grabbing
+To initiate a connection with telnet server and get any information about the target .
+
+
+> [!TIP]
+> The $TARGET_PORT is optional, but the default port is 23
+
+
+#### CLI tool
+::: tabs
+
+=== Unix-like
+
+```bash
+#Using Netcat
+nc -nv $TARGET_IP $TARGET_PORT
+#Using Telnet
+telnet $TARGET_IP $TARGET_PORT
+#Using Shodan-cli
+shodan stream --ports 23,1023,2323 --datadir telnet-data/ --limit 10000
+#Using nmap
+nmap -p  $TARGET_PORT -sVC  --script "*telnet* and safe" $TARGET_IP
+```
+
+
+=== Windows
+
+```powershell
+nc.exe -nv $TARGET_IP $TARGET_PORT
+telnet $TARGET_IP $TARGET_PORT
+nmap -p  $TARGET_PORT   --script telnet-ntlm-info.nse $TARGET_IP
+```
+:::
+
+#### Automated tools
+The following commands is typed in msfconsole (The Cli for Metasploit Framework)
+
+::: tabs
+
+=== Metasploit 
+
+```
+msf > use auxiliary/scanner/telnet/telnet_version
+msf > set rhosts $TARGET_IP 
+msf > set rport $TARGET_PORT
+msf > set threads 5
+msf > exploit
+```
+:::
+
+
+### Attacks 
+
+#### Passwordless Authentication
+
+Telnet allows users to connect to a server without needing a specific identity by utilizing a passwordless login feature. This method is commonly employed for accessing or downloading public files.
+
+To connect without a password, you would use the following command:
+
+```bash
+telnet $TARGET_IP
+
+#provide username
+#but not provide any password
+```
+#### Common Credentials
+
+If anonymous login is disabled on the Telnet server, trying common usernames and passwords like admin, administrator, root, user, or test can be a good initial step. This approach is less aggressive than attempting to guess passwords through brute force and is recommended to try first when accessing a server.
+
+```bash
+telnet $TARGET_IP
+
+#provide a common username
+#provide a common password
+```
+
+
+#### Brute Force
+
+::: tabs
+
+=== Nmap
+
+```bash
+nmap -p 23 --script telnet-brute $TARGET_IP
+```
+
+=== Hydra
+
+```bash
+hydra [-L users.txt or -l user_name] [-P pass.txt or -p password] -f [-S $TARGET_PORT] telnet://$TARGET_IP 
+
+hydra -l root -P $PATH_TO/seclists/Passwords/Default-Credentials/telnet-betterdefaultpasslist.txt $TARGET_IP telnet
+```
+
+The following commands is typed in msfconsole.
```

Comment:
Please remove this sentence, or put it right under the tab name, or else it will be in the preceding tab